Python Part/lib/fnn.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+from lib.sigmoid import sigmoid
+from utils.report_generator import generate_report
+
+
+class fnn_layers:
```

Comment:
what does fnn stands for?

Reply:
fully connected Neural Network, even tho it's deep but the you get my point !

Reply:
fully connected Neural Network, even tho it's deep but the you get my point !

Reply:
I don't get it.